README.md

```diff
@@ -1,35 +1,89 @@
 # podannotator
-Annotate pods with PVC in a given list of namepsaces to be backed up by velero/restic
+Annotates all volumes (with a PVC), in all pods, in a given list of namepsaces to be backed up by velero/restic.
 
+Namespace 1
+        -- Pod 1
+                -- Volume 1 (N1P1V1)
+                -- Volume 2 (N1P1V2)
+        -- Pod 2
+                -- Volume 1 (N1P2V1)
+                -- Volume 2 (N1P2V2)
+Namespace 2
+        -- Etc.
+
+Note: All pods within a given namespace will be backed up, if they have a PVC.  (Ignoring PVC's would include secrets.)  There is currently no requirement to be able to exclude volumes from backups.  See @todo in class if this is ever in scope.
 
 # Usage  
 
-        annotator namespace1 namespace2
+python3 annotator.py namespace1,namespace2
 
```

Comment:
can you add the requierment 
`pip3 install prettytable`
thanks :)

Reply:
Sorry, I don't understand where you mean?

Reply:
@zakkg3 Any chance you could review this for me please so we can get it merged (or fixed if more fixes are required?)

Thanks :)